chord_metadata_service/phenopackets/migrations/0010_v7_0_3.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.11 on 2024-03-26 20:10
```

Comment:
upcoming katsu release is 8.0.0 assuming the cBioPortal / auth changes make it in for v16.